mobile/lib/ui/viewer/actions/file_selection_actions_widget.dart

```diff
@@ -646,6 +691,8 @@ class _FileSelectionActionsWidgetState
     );
     if (actionResult?.action != null) {
       if (actionResult!.action == ButtonAction.first) {
+        generatedPathNotifier.value =
+            await _selectedFilePlaceholderPath(ownedSelectedFiles);
         await _copyLink();
```

Comment:
Should rename this method to something like `_sendLink()`. And do we need to still copy the link to clipboard for the sending the link with the placeholder? (`_copyLink` is copying the link to placeholder now)